templates/front/tab.member.tpl

```diff
@@ -1,7 +1,7 @@
 {if 'view_member' == $core.page.name}
-	{capture append='tabs_content' name='wall_posts'}
-		<div class="ia-wrap">
-			{include file="modules/wall/templates/front/block.wall.tpl" isView=true}
-		</div>
-	{/capture}
+    {capture append='tabs_content' name='wall_posts'}
+        <div class="ia-wrap">
+            {include file="modules/wall/templates/front/block.wall.tpl" isView=true}
```

Comment:
review please. it's better to include file according to our standard scheme, like module:wall/ etc.